### DISCUSSION TITLE
Add attributes to logs

### DISCUSSION THREAD

**Discussion by Flooze-IT** [open-telemetry/opentelemetry-java#4982] [Q&A]
Hi everyone,

i ask a question here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/7303#discussion-4596969 but i think the right place is here.

Sorry and many thanks for your help

**Comment by jkwatson** [comment]
Important: https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/7303#discussioncomment-4230152

**Reply by greatvovan** [reply]
That's a great suggestion, I also would like to set custom dynamic attributes with my log records but currently there seem to be no way. My current (clumsy) alternative is sending events within trace spans as events accept `attributes` parameter.
